### PR TITLE
tools/dm: batch updates in sync with docs-cn

### PR DESCRIPTION
**Pull request by dcalvin**
Batch updates in sync with docs-cn:
- **data-synchronization-features.md** and **shard-merge-scenario.md**: use _binlog event filter_ for the feature name
- **deployment.md**: update DM-Ansible download info
- **query-status.md** remove redundant code lines

### CODE REVIEW

File: tools/dm/query-status.md [hunk 1]
```diff
@@ -109,35 +109,6 @@ This document introduces the query result and subtask status of Data Migration (
                 "result": null
             }
         },
-        {
```
Comment (lilin90) [inline tools/dm/query-status.md]: I see that this section is in [docs-cn](https://pingcap.com/docs/tools/dm/query-status/). Are you sure that we need to remove it?

Comment (dcalvin) [reply tools/dm/query-status.md]: This section appeared twice, so we need to remove one occurence.

Comment (lilin90) [reply tools/dm/query-status.md]: Got it!

File: tools/dm/query-status.md [hunk 2]
```diff
@@ -109,35 +109,6 @@ This document introduces the query result and subtask status of Data Migration (
...
-                "masterBinlogGtid": "c0149e17-dff1-11e8-b6a8-0242ac110004:1-96",
-                "relaySubDir": "c0149e17-dff1-11e8-b6a8-0242ac110004.000001",
-                "relayBinlog": "(bin.000001, 28507)",
-                "relayBinlogGtid": "c0149e17-dff1-11e8-b6a8-0242ac110004:1-96",
-                "relayCatchUpMaster": true,
-                "stage": "Running",
-                "result": null
-            }
-        }
                 {
```
Comment (lilin90) [inline tools/dm/query-status.md]:
```suggestion
        {
```

File: tools/dm/query-status.md (hunk 1 above)
Comment (lilin90) [reply tools/dm/query-status.md]: Got it!

File: tools/dm/query-status.md [hunk 3]
```diff
@@ -108,37 +108,8 @@ This document introduces the query result and subtask status of Data Migration (
...
-                "relaySubDir": "c0149e17-dff1-11e8-b6a8-0242ac110004.000001",
-                "relayBinlog": "(bin.000001, 28507)",
-                "relayBinlogGtid": "c0149e17-dff1-11e8-b6a8-0242ac110004:1-96",
-                "relayCatchUpMaster": true,
-                "stage": "Running",
-                "result": null
-            }
-        }
-                {
+         },
```
Comment (lilin90) [inline tools/dm/query-status.md]:
```suggestion
        },
```

File: tools/dm/query-status.md [hunk 4]
```diff
@@ -108,37 +108,8 @@ This document introduces the query result and subtask status of Data Migration (
...
+         {
```
Comment (lilin90) [inline tools/dm/query-status.md]:
```suggestion
        {
```

**Comment by lilin90** [comment]
@csuzhangxc PTAL

**Pull request merged**